### ISSUE TITLE
[v2] Request: when dragging 'schedule' at the left/right border, auto-jump to prev/next period

### ISSUE THREAD

**Issue by mariusa** [nhn/tui.calendar#433]
Hi,

Need: re-schedule an event by drag & drop.
This works fine in week view if the new desired time is same week, but it doesn't work when wanting to re-schedule in 2 weeks from now.

Solution: when dragging the event/'schedule' and reaching the left/right border, auto-jump to prev/next calendar period (similar to ```calendar.invoke('next')```)

What do you think?

**Comment by jungeun-cho** [comment]
@mariusa 

Thank you for your idea.
Is it really convenient to move the calendar while people drag the event? 
I need a discussion inside our team.

**Comment by mariusa** [comment]
> Is it really convenient to move the calendar while people drag the event?

Yes! This came as a bug report from one of our users. It's very convenient for users to drag a schedule from Tuesday to Friday. Why not drag from Friday to next Monday?

**Comment by jungeun-cho** [comment]
> drag from Friday to next Monday

Oh!

Hmm... If I implement it, I need to define the details.
For example, it only supports weekly views....

**Comment by mariusa** [comment]
Right, only for the weekly view.

**Comment by qixeo** [comment]
Yes! This is exactly what I need. Or even the ability to change the offset from 7 to say, 5 or 3. Any word on this feature?

**Comment by adhrinae** [comment]
@qixeo 

Do you mean like [this](https://github.com/nhn/tui.calendar/issues/892)?

About this feature request, there's no plan to implement it yet.

**Comment by qixeo** [comment]
> @qixeo
> 
> Do you mean like [this](https://github.com/nhn/tui.calendar/issues/892)?
> 
> About this feature request, there's no plan to implement it yet.

In week view, a quick way to drag a task into the next week. Right now, for example, if I have a task on Sunday, I can't drag it to the next day (Monday). Since I depend on scheduling a task based on how many tasks are already in a day, it would be great to be able to drag the task to the far right and have the days or weeks slowly scroll until I find an empty day in which to drop it. 

The calendar is really great, but of course everyone always has a need for more features!

**Comment by adhrinae** [comment]
> even the ability to change the offset from 7 to say, 5 or 3.
I misunderstood this point is related to #892. Now I got the point.

I'll consider that feature after releasing v2. But I don't think this feature is generally needed.
